protostar-rust/src/cheatcodes_hint_processor.rs

```diff
@@ -303,43 +361,26 @@ fn execute_cheatcode_hint(
             let tx_result = account_tx
                 .execute(blockifier_state, block_context)
                 .expect("Failed to execute declare transaction");
+            // result_segment.
+            let felt_class_hash = felt252_from_hex_string(&class_hash.to_string()).unwrap();
 
-            insert_value_to_cellref!(
-                vm,
-                result,
-                felt252_from_hex_string(&class_hash.to_string()).unwrap()
-            )?;
             // TODO https://github.com/software-mansion/protostar/issues/2024
-            //  in case of errors above, consider not panicking, set an error and return it here
-            //  instead
-            insert_value_to_cellref!(vm, err_code, Felt252::from(0))?;
-            Ok(())
+            //  in case of errors above, consider not panicking,
+            //  set an error and return it here instead
+            insert_at_pointer(vm, &mut result_segment_ptr, Felt252::from(0)).unwrap();
+            insert_at_pointer(vm, &mut result_segment_ptr, felt_class_hash).unwrap();
```

Comment:
pls do not use unwraps

Reply:
This should be addressed in separate PR that should be rebased on top of this one: https://github.com/software-mansion/protostar/pull/2152
I will this comment there

Reply:
https://github.com/software-mansion/protostar/pull/2152#issuecomment-1620185539